```
2023-12-01 11:12:26 ======================================================================
2023-12-01 11:12:26 ERROR: test_cpu (test_neg_op.TestNegOpInt8)
2023-12-01 11:12:26 ----------------------------------------------------------------------
2023-12-01 11:12:26 Traceback (most recent call last):
2023-12-01 11:12:26   File "/workspace/Paddle/build/test/legacy_test/test_neg_op.py", line 59, in test_cpu
2023-12-01 11:12:26     self.run_static()
2023-12-01 11:12:26   File "/workspace/Paddle/build/python/paddle/pir_utils.py", line 115, in impl
2023-12-01 11:12:26     func(*args, **kwargs)
2023-12-01 11:12:26   File "/workspace/Paddle/build/test/legacy_test/test_neg_op.py", line 49, in run_static
2023-12-01 11:12:26     st_result = exe.run(feed={"input": self.input}, fetch_list=[result])
2023-12-01 11:12:26   File "/workspace/Paddle/build/python/paddle/base/executor.py", line 1712, in run
2023-12-01 11:12:26     res = self._run_pir_impl(
2023-12-01 11:12:26   File "/workspace/Paddle/build/python/paddle/base/executor.py", line 2029, in _run_pir_impl
2023-12-01 11:12:26     self._pir_feed_data(program, feed, scope)
2023-12-01 11:12:26   File "/workspace/Paddle/build/python/paddle/base/executor.py", line 1336, in _pir_feed_data
2023-12-01 11:12:26     pir_check_feed_shape_type(
2023-12-01 11:12:26   File "/workspace/Paddle/build/python/paddle/base/executor.py", line 317, in pir_check_feed_shape_type
2023-12-01 11:12:26     raise ValueError(
2023-12-01 11:12:26 ValueError: The data type of fed Variable 'input' must be 'float64', but received 'int8'
```
这个是要改yaml或者kernel的配置文件吗?

--- a/python/paddle/tensor/logic.py
+++ b/python/paddle/tensor/logic.py
@@ -1108,7 +1108,7 @@ def is_tensor(x):
     """
     if in_dynamic_or_pir_mode():
         return isinstance(
-            x, (Tensor, paddle.base.core.eager.Tensor, paddle.pir.OpResult)
+            x, (Tensor, paddle.base.core.eager.Tensor, paddle.pir.Value)
         )
     else:
         return isinstance(x, Variable)

--- a/python/paddle/tensor/manipulation.py
+++ b/python/paddle/tensor/manipulation.py
@@ -5596,7 +5596,7 @@ def unflatten(x, axis, shape, name=None):
         new_shape = (
             list(x.shape[:axis]) + list(shape) + list(x.shape[axis + 1 :])
         )
-    elif isinstance(shape, (Variable, paddle.pir.OpResult)):
+    elif isinstance(shape, (Variable, paddle.pir.Value)):
         # The data type returned by `paddle.shape` is only 'int32'.
         new_shape = paddle.concat(
             [

--- a/python/paddle/tensor/stat.py
+++ b/python/paddle/tensor/stat.py
@@ -314,7 +314,7 @@ def nanmedian(x, axis=None, keepdim=False, name=None):
             >>> print(y4.numpy())
             2.0
     """
-    if not isinstance(x, (Variable, paddle.pir.OpResult)):
+    if not isinstance(x, (Variable, paddle.pir.Value)):
         raise TypeError("In median, the input x should be a Tensor.")
 
     if isinstance(axis, (list, tuple)) and len(axis) == 0:
@@ -404,7 +404,7 @@ def median(x, axis=None, keepdim=False, name=None):
             [[4., 5., 6., 7.]])
 
     """
-    if not isinstance(x, (Variable, paddle.pir.OpResult)):
+    if not isinstance(x, (Variable, paddle.pir.Value)):
         raise TypeError("In median, the input x should be a Tensor.")
 
     if in_dynamic_mode() and x.size == 0:

--- a/python/paddle/tensor/stat.py
+++ b/python/paddle/tensor/stat.py
@@ -407,7 +407,8 @@ def median(x, axis=None, keepdim=False, name=None):
     if not isinstance(x, (Variable, paddle.pir.OpResult)):
         raise TypeError("In median, the input x should be a Tensor.")
 
-    if x.size == 0:
+    if in_dynamic_mode() and x.size == 0:
+        # TODO: Currently, `__eq__` don't support arguments (`pir.Value` & `int`)
         raise ValueError("In median, the size of input x should not be 0.")
 
     is_flatten = False

--- a/test/legacy_test/test_neg_op.py
+++ b/test/legacy_test/test_neg_op.py
@@ -38,25 +38,26 @@ class TestNegOp(unittest.TestCase):
 
     @test_with_pir_api
     def run_static(self, use_gpu=False):
-        input = paddle.static.data(
-            name='input', shape=[32, 8], dtype=self.dtype
-        )
-        result = paddle.neg(input)
+        with paddle.static.program_guard(paddle.static.Program()):
+            input = paddle.static.data(
+                name='input', shape=[32, 8], dtype=self.dtype
+            )
+            result = paddle.neg(input)
 
-        place = paddle.CUDAPlace(0) if use_gpu else paddle.CPUPlace()
-        exe = paddle.static.Executor(place)
-        exe.run(paddle.static.default_startup_program())
-        st_result = exe.run(feed={"input": self.input}, fetch_list=[result])
-        expected_result = np.negative(self.input)
-        np.testing.assert_allclose(st_result[0], expected_result, rtol=1e-05)
+            place = paddle.CUDAPlace(0) if use_gpu else paddle.CPUPlace()
+            exe = paddle.static.Executor(place)
+            exe.run(paddle.static.default_startup_program())
+            st_result = exe.run(feed={"input": self.input}, fetch_list=[result])
+            expected_result = np.negative(self.input)
+            np.testing.assert_allclose(
+                st_result[0], expected_result, rtol=1e-05
+            )
 
     def test_cpu(self):
         paddle.disable_static(place=paddle.CPUPlace())
         self.run_imperative()
         paddle.enable_static()
-
-        with paddle.static.program_guard(paddle.static.Program()):
-            self.run_static()
+        self.run_static()
 
     def test_gpu(self):
         if not paddle.base.core.is_compiled_with_cuda():
@@ -65,9 +66,7 @@ class TestNegOp(unittest.TestCase):
         paddle.disable_static(place=paddle.CUDAPlace(0))
         self.run_imperative()
         paddle.enable_static()
-
-        with paddle.static.program_guard(paddle.static.Program()):
-            self.run_static(use_gpu=True)
+        self.run_static(use_gpu=True)
 
 
 class TestNegOpFp32(TestNegOp):

